parl/utils/tests/logger_test.py

```diff
@@ -0,0 +1,27 @@
+#-*- coding: utf-8 -*-
+#File: test_logging.py
+
+from parl.utils import logger
+import threading as th
+
```

Comment:
All test codes must be written in a unit test class.
You can refer other unit test codes.